model/registry/azure.yaml

```diff
@@ -0,0 +1,100 @@
+groups:
+  - id: registry.azure.sdk
+    prefix: az
+    type: attribute_group
+    brief: >
+      This document defines generic attributes for Azure SDK.
+    attributes:
+      - id: service_request_id
+        type: string
+        stability: experimental
+        brief: "The unique reference string that identifies a request"
+        examples:
+          - "00000000-0000-0000-0000-000000000000"
+  - id: registry.azure.logs
+    prefix: azure
```

Comment:
```suggestion
    prefix: az
```

Reply:
There are two proposed azure prefixes, `az` is used by the Azure SDK and `azure` by the Azure logs.

Reply:
Unfortunately Azure SDK `az` ship has sailed, we shipped some stable libraries that emit attributes with this prefix and there are integrations that depend on it.

If Azure services would start to emit otel-compatible telemetry, I'd recommend them to use `az` as well, for the sake of consistency. 

So, unless there are strong reasons to use `azure` and not `az`,  we should be using the same prefix consistently for different signals.

Reply:
Okay, I thought we were already seeing telemetry with both `az` and `azure` prefixes. I don't see a strong need to have both, so will move everything under the `az` prefix for now. We can look to re-introduce the `azure` prefix later if needed.